Directory.Build.props

```
@@ -24,8 +24,9 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies> <!-- required due to https://github.com/tonerdo/coverlet/issues/72 -->
     <CoverletOutputFormat>opencover</CoverletOutputFormat>
     <Include>[System.Windows.*]*</Include>
-    <!-- Exclude only Obsolete and tagged with ExcludeFromCodeCoverage !!Avoid using this!! -->
-    <Exclude />
+    <!-- Exclude dependencies that embed their symbol information -->
+    <Exclude>Microsoft.DotNet.XUnitExtensions</Exclude>
```

I *think* this is the form we need:

```suggestion
    <Exclude>[Microsoft.DotNet.XUnitExtensions]*;[xunit*]*</Exclude>
```